### DISCUSSION TITLE
Does support portable version ?

### DISCUSSION THREAD

**Discussion by jimmy123j** [massCodeIO/massCode#476] [General]
'cause can't install S/W at company, hope can support the portable version

**Comment by SadSock** [comment]
Many companies don't allow arbitrary software installations on office computers, only portable versions can be used. Supporting portable applications could help promote software distribution, as it enables usage even on restricted work computers.

**Comment by Airr** [comment]
In case y'all were unaware, for the Windows installer you can open it in the 7zip GUI, go inside the '$PLUGINSDIR', double-click the 'app-64.7z' file, and drag the contents out to a folder in your home folder.  

Then just create a shortcut to the 'massCode.exe' file, and you're done.

You can do this with a lot of Electron-based apps that use an exe installer.

HTH!

AIR.

**Reply by jimmy123j** [reply]
thanks for help, already successed !!!

**Comment by antonreshetov** [comment]
In version 4, I will try to add a portable version for Windows.
